nautobot_circuit_maintenance/views.py

```diff
@@ -1,20 +1,197 @@
 """Views for Circuit Maintenance."""
+from datetime import datetime as datet
+import datetime
 import logging
 
 import google_auth_oauthlib
 
+from django.conf import settings
 from django.shortcuts import get_object_or_404, render, redirect
 from django.urls import reverse
 from django.urls.exceptions import NoReverseMatch
 from nautobot.core.views import generic
-from nautobot.circuits.models import Provider
+from nautobot.circuits.models import Circuit, Provider
 from nautobot_circuit_maintenance import filters, forms, models, tables
 from nautobot_circuit_maintenance.handle_notifications.sources import RedirectAuthorize, Source
+from nautobot_circuit_maintenance.models import CircuitMaintenance
 
 
 logger = logging.getLogger(__name__)
 
 
+class CircuitMaintenanceOverview(generic.ObjectListView):
+    """View for an overview dashboard of summary view."""
```

Comment:
I would appreciate a summary of the info provided in this overview (the report keys)

Reply:
Help me out on this one, are we talking what we are passing into the templates?

Reply:
I was thinking on a summary of the information that this view is providing, in a bullet point list. So, then you explore the code with some context.

Reply:
Shoot, I need to do this yet.